Source/Core/DolphinQt/Translation.cpp

```diff
@@ -275,7 +275,7 @@ static bool TryInstallTranslator(const QString& exact_language_code)
     std::string lang = qlang.toStdString();
     auto filename =
 #if defined _WIN32
-        File::GetExeDirectory() + StringFromFormat("/Languages/%s/dolphin-emu.mo", lang.c_str())
+        File::GetExeDirectory() + StringFromFormat("/Languages/%s.mo", lang.c_str())
```

Comment:
Was this change done for any particular reason?

Reply:
Not really, just that I thought removing a level of directories was a clean up

Reply:
I think the intention of the old style of paths is that you could have multiple mo files per language for different components of the software... Which we never ended up using and probably won't have any reason to use in the future either, so I suppose either way is fine here.

Reply:
Yea that was my thinking too